### PR TITLE
Adding a no-js message

### PR DESCRIPTION
**Pull request by Guntrisoft**
screen shot 2016-12-23 at 15 17 56

- hiding search and selection form elements from nojs browsers
- adding enable JS message for nojs browsers across various pages

### CODE REVIEW

File: app/views/shared/_nojs.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<noscript>
+<div class="alert alert-success" role="alert">
```
Comment (benlovell) [inline app/views/shared/_nojs.html.erb]: Identation is a bit off here 😱 😉

File: app/views/shared/_nojs.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  <p>Please ensure that javascript is enabled on your browser.</p>
```
Comment (benlovell) [inline app/views/shared/_nojs.html.erb]: That should be 'JavaScript' right?

File: app/views/shared/_nojs.html.erb (hunk 1 above)
Comment (Guntrisoft) [inline app/views/shared/_nojs.html.erb]: fixed

File: app/views/shared/_nojs.html.erb (hunk 2 above)
Comment (Guntrisoft) [inline app/views/shared/_nojs.html.erb]: Correct! - fixed

**Pull request merged**